samples/drawing/drawing.cpp

```diff
@@ -2535,7 +2535,8 @@ void MyFrame::OnSave(wxCommandEvent& WXUNUSED(event))
         else
 #endif // wxUSE_POSTSCRIPT
         {
-            wxBitmap bmp(width, height);
+            wxBitmap bmp;
+            bmp.CreateWithDIPSize(wxSize(width, height), GetDPIScaleFactor());
```

Comment:
This causes the image size to scale as well, see final comment.

Reply:
Yes, but I think this is the right thing to do, i.e. at 200% the image should be twice as big as at 100%, shouldn't it?

Reply:
Yeah, that sounds right. When/if the drawing sample is modified to use `FromDIP` for all the coordinates and sizes it should be correct.